### PR TITLE
Save data to local storage

### PR DESCRIPTION
**Pull request by fauziyahzeeyah**
-  Add formData to local storage
-  Update styles
-  Corrected ESlint errors

### CODE REVIEW

File: js/local-storage.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+});
+
+window.addEventListener('load', () => {
+  const storedData = localStorage.getItem('portfolio-form-data');
+  formData = JSON.parse(storedData);
+
+  fields.forEach((field) => {
+    field.value = formData[field.name];
+  });
+});
```
Comment (Favourezeugwa) [inline js/local-storage.js:23]: - I don't think data entered in your form is being saved to the local storage.  Can we fix this?? 🤗 

Here is what i could see on my browser after cloning your repo 👇🏽 

empt

**Review by Favourezeugwa — COMMENTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @fauziyahzeeyah , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-brianna


## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/local-storage.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+function saveData() {
+  let name = '';
+  let email = '';
+  let message = '';
```
Comment (tufoinnkuo10) [inline js/local-storage.js:4]: - [ ] great job. Please ensure that After filling the form, and not submitting it, the entered values should be saved in the browser. When the user loads the page, the code checks if there's any saved data in the local storage and if so the input fields are pre-filled with the data.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Team
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by binhussen — APPROVED** [review]
Hi @fauziyahzeeyah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**